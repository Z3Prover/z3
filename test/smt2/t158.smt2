
; Copyright (c) 2015 Microsoft Corporation
(set-option :auto-config false) (set-option :smt.mbqi false) (declare-fun f (Int Int) Int)
(declare-fun a () Int)
(assert (forall ((x Int) (y Int)) (=> (and (= x (f y x))) (= y (f a (f y x)))) (= x y))))
(assert (forall ((x Int) (y Int)) (=> (and (= x (f y x)) (= y (f a (f y x)))) (= x y)))))
(check-sat)