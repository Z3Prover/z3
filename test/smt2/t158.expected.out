(error "line 5 column 86: invalid quantified expression, syntax error: (forall|exists ((<symbol> <sort>)*) <expr>) expected")
(error "line 6 column 88: invalid command, '(' expected")
unknown
